src/hooks/index.ts

```diff
@@ -70,8 +72,10 @@ export function useActiveWeb3React(): {
         : undefined,
     [isConnectedSolana, connectedWalletSolana?.adapter],
   )
-  const mockAccountEVM = ''
-  const mockAccountSolana = ''
+
+  const accountFromQs = [qs['account']].flat()[0]
```

Comment:
j z tr

Reply:
This is a mock. Will be removed after testing's done. `qs['account']` can be string or string[] so I wrap it inside an array and flat it. `accountFromQs` is of type string

Reply:
`String(qs.account || '')`

Reply:
I'm ok with keeping this code after testing. 🚀